### PR TITLE
Gitlab cli Updated for version 1.48.0

### PR DESCRIPTION
**Pull request by DinizBortolotto**

### CODE REVIEW

**Comment by willysr** [comment]
@sbo-bot: build development/gitlab-cli

**Comment by willysr** [comment]
@DinizBortolotto this failed to build

**Comment by DinizBortolotto** [comment]
Gitlab changed their goreleaser filename template so we needed to fix SlackBuild.
See [https://gitlab.com/gitlab-org/cli/-/merge_requests/1701](url)

File: development/gitlab-cli/gitlab-cli.SlackBuild [hunk 1]
```diff
@@ -26,30 +26,26 @@ cd $(dirname $0) ; CWD=$(pwd)
...
 PRGNAM=gitlab-cli
 DEBNAM=glab
-VERSION=${VERSION:-1.46.1}
+DEBARCH=amd64
+VERSION=${VERSION:-1.48.0}
 BUILD=${BUILD:-1}
 TAG=${TAG:-_SBo}
 PKGTYPE=${PKGTYPE:-tgz}
 
 if [ -z "$ARCH" ]; then
```
Comment (aclemons) [inline development/gitlab-cli/gitlab-cli.SlackBuild:35]: This doesn't work if someone has ARCH already set in the env.

The previous code which checked for an empty ARCH, then set it. And then a separate if block built the DEBARCH var based on it was correct.

Could you revert that change?

**Comment by willysr** [comment]
@sbo-bot: build development/gitlab-cli

**Comment by willysr** [comment]
this solves for x86_64 only, so the best option is to restore the ARCH check like what @aclemons suggested

**Comment by willysr** [comment]
@sbo-bot: build development/gitlab-cli

**Comment by willysr** [comment]
merged

**Comment by DinizBortolotto** [comment]
> merged

Tks for your support and sorry for the mess in git log. Im still learning that.

**Pull request closed without merging**